src/async-wrap.cc

```diff
@@ -468,6 +468,13 @@ void AsyncWrap::QueueDestroyId(const FunctionCallbackInfo<Value>& args) {
   PushBackDestroyId(Environment::GetCurrent(args), args[0]->NumberValue());
 }
 
+void AsyncWrap::AddWrapMethods(Environment* env,
```

Comment:
On third thought `AsyncHooks` is not good, although that's the "context", the methods are only used by the hooks, and are not _the_ hooks. so:
+2 for `AsyncIDs`
+1 for `AsyncWrap`